.github/workflows/main.yml

```diff
@@ -0,0 +1,61 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  build:
+
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest]
+        python-version: [2.7, 3.5, 3.7, 3.8]
```

Comment:
Is this `python-version` required if we're explicit below in `include`?

Reply:
Yes - the include is on the version specified I believe. https://help.github.com/en/actions/reference/workflow-syntax-for-github-actions#example-including-configurations-in-a-matrix-build. My understanding is that you're augmenting and existing build.